README.md

```diff
@@ -1,24 +1,33 @@
+Ansible Roles, Playbooks (and Tower Configuration metadata) for installing, configuring and interacting with the Red Hat Satellite. The roles will be created using [foreman-ansible-modules] where possible(https://github.com/theforeman/foreman-ansible-modules) (FAM).
 
-## Rules for contributing Roles to this project
+# Contribution Guidelines
 
-* The use cases supported by the roles should be beneficial for many users.
-* Input variables externalized in `defaults` folder of roles for easy reuse.
-* The roles should be compatible with the currently (upstream) supported Ansible versions, 2.7+ at the time of writing.
-* The roles should be compatible with any Python supported by Ansible upstream (2.7, 3.5, 3.6, 3.7 at the time of writing), however, roles usually don’t require any special Python features (in contrast to modules).
-* The roles should be compatible with the latest released version of the [FAM](https://github.com/theforeman/foreman-ansible-modules) collection (0.3.0 at the time of writing). While defining a dependency on an older FAM release is possible, it is discouraged.
-* The roles shall not use hammer via command/shell or the uri module to talk to the API directly.
+- This collection is a collaborative development environment.
+- This collection may not accept all Ansible modules proposed. If you are writing a new module related to Foreman, consider contributing it to [FAM](https://github.com/theforeman/foreman-ansible-modules). For modules that have other source projects, we may request you to open a pull request against respective projects.
+
+
+## Roles
+
+* The roles written in this collection should be extensible, reusable and idempotent.
+* The input variables should be externalized in the `defaults` directory of the roles allowing for easy reuse.
+* The variables that are required by the role but not required by user to be modified, should reside in `vars` directory.
+* The roles should be compatible with the current (upstream) supported Ansible versions, 2.9+(?) at the time of writing.
+* The roles should be compatible with any Python supported by Ansible upstream (2.7(?), 3.5, 3.6, 3.7 at the time of writing), however, roles usually don’t require any special Python features (in contrast to modules).
+* The roles should be compatible with the latest released version of the [FAM](https://github.com/theforeman/foreman-ansible-modules) collection (v1.0.1 at the time of writing). While defining a dependency on an older FAM release is possible, it is discouraged.
+* The roles should attempt to not use `command`, `shell`, or `raw` Ansible modules unless there is no other solution.
+* The roles should not use hammer via command/shell or the uri module to talk to the API directly.
```

Comment:
+"unless there is no other solution"? We don't have 100% coverage with FAM, so some things are only possible with hammer *today*.

Reply:
I think at that point its fine since its an initial commit and we can add and document times we use those modules.

Reply:
Agreed. We want to avoid it where we can. But if no other solution, then shell etc. modules are ok. That's what we intend to say in the docs.